samwise.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?